src/js/edu-benefits/1995/components/IntroductionPage.jsx

```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
 import ProgressButton from '../../../common/components/form-elements/ProgressButton';
 
 class IntroductionPage extends React.Component {
-  constructor() {
-    super();
-    this.goForward = this.goForward.bind(this);
-  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
-  goForward() {
+  goForward = () => {
```

Comment:
We can do this rather than `this.goForward = this.goForward.bind(this);` in the `super()` to save space. Huzzah for ES6!

If the other way was used for a reason, just let me know.